   237.3ms (setup:799): 2024-04-05 12:11:56
   237.4ms (setup:800): /usr/bin/pipx run python -c import matplotlib; print(matplotlib.__version__)
   237.5ms (setup:801): pipx version is 1.2.0
   237.5ms (setup:802): Default python interpreter is '/usr/bin/python3'
   241.0ms (run:106): venv location is /home/ngallego/.local/pipx/.cache/9d897ec88618d10
   241.6ms (run_subprocess:173): running /usr/bin/python3 -m venv --without-pip /home/ngallego/.local/pipx/.cache/9d897ec88618d10
   302.1ms (run_subprocess:186): stdout:
   302.2ms (run_subprocess:188): stderr:
   302.3ms (run_subprocess:189): returncode: 0
   303.9ms (run_subprocess:173): running /home/ngallego/.local/pipx/.cache/9d897ec88618d10/bin/python -c import sysconfig; print(sysconfig.get_path('purelib'))
   321.3ms (run_subprocess:186): stdout: /home/ngallego/.local/pipx/.cache/9d897ec88618d10/lib/python3.11/site-packages
   321.4ms (run_subprocess:189): returncode: 0
   321.6ms (run_subprocess:173): running /home/ngallego/.local/pipx/shared/bin/python -c import sysconfig; print(sysconfig.get_path('purelib'))
   345.5ms (run_subprocess:186): stdout: /home/ngallego/.local/pipx/shared/lib/python3.11/site-packages
   345.7ms (run_subprocess:189): returncode: 0
   346.1ms (run_subprocess:173): running /home/ngallego/.local/pipx/.cache/9d897ec88618d10/bin/python --version
   348.7ms (run_subprocess:186): stdout: Python 3.11.6
   348.8ms (run_subprocess:188): stderr:
   348.9ms (run_subprocess:189): returncode: 0
   349.1ms (package_name_from_spec:322): Determined package name: python
   349.1ms (package_name_from_spec:323): Package name determined in 0.0s
   349.3ms (_parsed_package_to_package_or_url:147): cleaned package spec: python
   349.8ms (run_subprocess:173): running /home/ngallego/.local/pipx/.cache/9d897ec88618d10/bin/python -m pip install python
  1401.1ms (run_subprocess:189): returncode: 1
  1401.7ms (subprocess_post_check_handle_pip_error:335): '/home/ngallego/.local/pipx/.cache/9d897ec88618d10/bin/python -m pip install python' failed
  1402.1ms (subprocess_post_check_handle_pip_error:352): Fatal error from pip prevented installation. Full pip output in file:
    /home/ngallego/.local/pipx/logs/cmd_2024-04-05_12.11.56_pip_errors.log
  1404.2ms (cli:866): PipxError: Error installing python.
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pipx/main.py", line 863, in cli
    return run_pipx_command(parsed_pipx_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pipx/main.py", line 200, in run_pipx_command
    commands.run(
  File "/usr/lib/python3/dist-packages/pipx/commands/run.py", line 107, in run
    _download_and_run(
  File "/usr/lib/python3/dist-packages/pipx/commands/run.py", line 143, in _download_and_run
    venv.install_package(
  File "/usr/lib/python3/dist-packages/pipx/venv.py", line 253, in install_package
    raise PipxError(
pipx.util.PipxError: Error installing python.
  1406.6ms (cli:874): pipx finished.
